src/components/pageControl/index.js

```diff
@@ -1,17 +1,19 @@
 import _ from 'lodash';
```

Comment:
The code looks great! very minimal and simple (:

Reply:
10x :)